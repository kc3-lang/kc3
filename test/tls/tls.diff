--- tls.out.expected	Fri Sep 12 16:14:59 2025
+++ tls.out	Fri Sep 12 16:15:10 2025
@@ -1 +1,10 @@
-
+require Socket
+true
+require Socket.Buf
+true
+require TLS
+true
+TLS.init()
+TLS
+config = TLS.Config.new()
+Segmentation fault (core dumped) 
--- tls.ret.expected	Fri Sep 12 16:15:05 2025
+++ tls.ret	Fri Sep 12 16:15:10 2025
@@ -1 +1 @@
-0
+139
