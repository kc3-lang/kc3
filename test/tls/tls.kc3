quote require Socket
require Socket
quote require Socket.Buf
require Socket.Buf
quote require TLS
require TLS
quote TLS.init()
TLS.init()
quote config = TLS.Config.new()
config = TLS.Config.new()
quote TLS.set_ca_cert_path(config, TLS.ca_cert_path())
TLS.set_ca_cert_path(config, TLS.ca_cert_path())
quote (server = Socket.listen("127.0.0.1", "57000")); void
(server = Socket.listen("127.0.0.1", "57000")) ; void
quote (client = Socket.Buf.connect("127.0.0.1", "57000")) ; void
(client = Socket.Buf.connect("127.0.0.1", "57000")) ; void
quote (server_client = Socket.Buf.accept(server)) ; void
(server_client = Socket.Buf.accept(server)) ; void
quote Socket.Buf.close(server_client)
Socket.Buf.close(server_client)
quote Socket.Buf.close(client)
Socket.Buf.close(client)
quote Socket.close(server)
Socket.close(server)
