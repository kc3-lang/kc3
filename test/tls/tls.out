require Socket
true
require Socket.Buf
true
require TLS
true
TLS.init()
TLS
config = TLS.Config.new()
Segmentation fault (core dumped) 
